src/Uno.UI.Wasm/tsBindings/WindowManagerSetPointerEventsParams.ts

```diff
@@ -0,0 +1,20 @@
+/* TSBindingsGenerator Generated code -- this code is regenerated on each build */
+class WindowManagerSetPointerEventsParams
+{
+	/* Pack=4 */
+	HtmlId : number;
+	Enabled : boolean;
+	public static unmarshal(pData:number) : WindowManagerSetPointerEventsParams
+	{
+		let ret = new WindowManagerSetPointerEventsParams();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'ret' is never reassigned; use 'const' instead of 'let'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5247895)